scenarios/action_recognition/00_webcam.ipynb

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
<p>[minor] not sure if the sentence is ending here "By default, our VideoLearner will load.."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done